users/cazfi.nix

```diff
@@ -8,7 +8,7 @@
       openssh.authorizedKeys.keys = [
         "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHzAww8Md+anrVfg93jNYey35Lu/YPEdbEh9QRu+riyf cazfi@cazfi-wlt"
       ];
-      extraGroups = ["wheel" "networkmanager"];
+      extraGroups = ["wheel" "networkmanager" "docker"];
```

Comment:
This is now in common part of ghaf-infra, not in prbuilder specific part. Does it work correctly for other hosts, which may not even have docker group at all?

Reply:
as far as I know this should be fine. the group is ignored if it does not exist.